webrender/src/device/gfx/command.rs

```diff
@@ -8,39 +8,36 @@ use hal::pool::RawCommandPool;
 pub struct CommandPool<B: hal::Backend> {
     command_pool: B::CommandPool,
     command_buffers: Vec<B::CommandBuffer>,
-    size: usize,
 }
 
 impl<B: hal::Backend> CommandPool<B> {
-    pub(super) fn new(mut command_pool: B::CommandPool) -> Self {
-        let command_buffer = command_pool.allocate_one(hal::command::RawLevel::Primary);
+    pub(super) fn new(command_pool: B::CommandPool) -> Self {
         CommandPool {
             command_pool,
-            command_buffers: vec![command_buffer],
-            size: 0,
+            command_buffers: vec![],
         }
     }
 
-    pub(super) fn acquire_command_buffer(
-        &mut self,
-    ) -> &mut B::CommandBuffer {
-        if self.size >= self.command_buffers.len() {
+    pub(super) fn acquire_command_buffer(&mut self) {
```

Comment:
We could rename this to something like `init_command_buffer` or `create_command_buffer`.